### ISSUE TITLE
Does not work on stock Google Pixel 4a 5G

### ISSUE THREAD

**Issue by BilliTheKid** [E7KMbb/Fake_5G_icon#6]
I am trying to install the mod via Magisk but it keeps saying Installation failed/not supported.

**Comment by wefalltomorrow** [comment]
Same issue here on a different device, but running an AOSP-based ROM. 

My SystemUI.apk is located in /Product/PRIV-APP/SystemUI, could that be why the script isn't compatible with my device?

**Comment by BilliTheKid** [comment]
> Same issue here on a different device, but running an AOSP-based ROM.
> 
> My SystemUI.apk is located in /Product/PRIV-APP/SystemUI, could that be why the script isn't compatible with my device?

Did you get it to work?

**Comment by wefalltomorrow** [comment]
> > Same issue here on a different device, but running an AOSP-based ROM.
> > My SystemUI.apk is located in /Product/PRIV-APP/SystemUI, could that be why the script isn't compatible with my device?
> 
> Did you get it to work?

Sadly not, I had a look through the script to try and redirect it to where my SystemUI is located but the code was a little too complex for me. Hopefuly the author sees this thread and makes a fix for it :)

**Comment by E7KMbb** [comment]
already fixed

**Comment by wefalltomorrow** [comment]
@E7KMbb Sadly it still occurs with v5

![Screenshot_20210209-203249](https://user-images.githubusercontent.com/45746576/107344049-eba6c000-6ab9-11eb-9683-baa630428735.jpg)

**Comment by BilliTheKid** [comment]
> already fixed

It has not been fixed. Still getting installation failed.
![Screenshot_20210209-094159](https://user-images.githubusercontent.com/56843472/107345125-2bba7280-6abb-11eb-8f89-90c6425a7e57.png)

**Comment by E7KMbb** [comment]
@wefalltomorrow Try v5.1

**Comment by E7KMbb** [comment]
@BilliTheKid Try v5.1

**Comment by wefalltomorrow** [comment]
@E7KMbb I get the same error sadly.

**Comment by BilliTheKid** [comment]
> @BilliTheKid Try v5.1

Still not working
![Screenshot_20210209-122231](https://user-images.githubusercontent.com/56843472/107363092-94144e80-6ad1-11eb-80d4-3126de7ceb2e.png)

**Comment by E7KMbb** [comment]
@BilliTheKid sh `find /system -name SystemUI*.apk` and tell me the result

**Comment by wefalltomorrow** [comment]
> @BilliTheKid sh `find /system -name SystemUI*.apk` and tell me the result

How do I run an 'sh' command?

**Comment by BilliTheKid** [comment]
> @BilliTheKid sh `find /system -name SystemUI*.apk` and tell me the result

Yes how do I run this command please?

**Comment by BilliTheKid** [comment]
> @BilliTheKid sh `find /system -name SystemUI*.apk` and tell me the result

![Screenshot_20210209-125556](https://user-images.githubusercontent.com/56843472/107366570-38988f80-6ad6-11eb-8aff-5ad7dbd7d1fc.png)

**Comment by wefalltomorrow** [comment]
You need to run SU first to get root privilege in terminal, although even then it doesn't list anything for me. I tried ADB as well and making an .sh file with the 'find' command in it and still nothing.

**Comment by E7KMbb** [comment]
> > @BilliTheKid sh`find /system -name SystemUI*.apk`告诉我结果
> 
> ![截图_20210209-125556](https://user-images.githubusercontent.com/56843472/107366570-38988f80-6ad6-11eb-8aff-5ad7dbd7d1fc.png)

![Screenshot_20210209-213141](https://user-images.githubusercontent.com/34418195/107370707-4b36b700-6b1e-11eb-9de5-fcea510522f8.png)

**Comment by wefalltomorrow** [comment]
Still nothing. 
![Screenshot_20210210-003939](https://user-images.githubusercontent.com/45746576/107371543-4bae5e00-6adc-11eb-8187-55538c5bda0e.jpg)

**Comment by BilliTheKid** [comment]
Yep nothing for me neither

**Comment by wefalltomorrow** [comment]
The reason nothing is found is per my first reply in this thread, my SystemUI.apk isn't located in /system. It's located here: /product/priv-app/SystemUI/SystemUI.apk

![Screenshot_20210210-004621](https://user-images.githubusercontent.com/45746576/107372346-384fc280-6add-11eb-8be0-409ffaf97a83.jpg)

**Comment by E7KMbb** [comment]
@wefalltomorrow Try to change `find /system` in customize.sh to `find / `  or  `/product`

**Comment by wefalltomorrow** [comment]
> @wefalltomorrow Try to change `find /system` in customize.sh to `find / ` or `/product`

That worked and flashed finally! But it doesn't change the icon for, I assume because I'm using a custom ROM with built-in customisation that modifies SystemUI. Anyhow, thanks for your all your work and support in trying to get this issue fixed. I doubt anything can be done with my current ROM so I'll just leave it at that.

![Screenshot_20210210-120336](https://user-images.githubusercontent.com/45746576/107451974-7eb82680-6b9c-11eb-8c14-65203818fd0d.jpg)

![Screenshot_20210210-120501](https://user-images.githubusercontent.com/45746576/107452041-a27b6c80-6b9c-11eb-8d8a-00e1f4720371.jpg)

![Screenshot_20210210-121106](https://user-images.githubusercontent.com/45746576/107452052-a5765d00-6b9c-11eb-853a-5f4bb0f01d8c.jpg)

![Screenshot_20210210-121113](https://user-images.githubusercontent.com/45746576/107452062-aa3b1100-6b9c-11eb-960e-1821de1bef45.jpg)

**Comment by E7KMbb** [comment]
> > @wefalltomorrow Try to change `find /system` in customize.sh to `find / ` or `/product`
> 
> That worked and flashed finally! But it doesn't change the icon for, I assume because I'm using a custom ROM with built-in customisation that modifies SystemUI. Anyhow, thanks for your all your work and support in trying to get this issue fixed. I doubt anything can be done with my current ROM so I'll just leave it at that.
> 
> ![Screenshot_20210210-120336](https://user-images.githubusercontent.com/45746576/107451974-7eb82680-6b9c-11eb-8c14-65203818fd0d.jpg)
> 
> ![Screenshot_20210210-120501](https://user-images.githubusercontent.com/45746576/107452041-a27b6c80-6b9c-11eb-8d8a-00e1f4720371.jpg)
> 
> ![Screenshot_20210210-121106](https://user-images.githubusercontent.com/45746576/107452052-a5765d00-6b9c-11eb-853a-5f4bb0f01d8c.jpg)
> 
> ![Screenshot_20210210-121113](https://user-images.githubusercontent.com/45746576/107452062-aa3b1100-6b9c-11eb-960e-1821de1bef45.jpg)

No, this is actually a problem with magisk. Magisk cannot mount directories other than the system partition. I think it cannot support it.

**Comment by wefalltomorrow** [comment]
Oh that's a shame, thanks for the response and help trying to get this to work.

**Comment by BilliTheKid** [comment]
> > @wefalltomorrow Try to change `find /system` in customize.sh to `find / ` or `/product`
> 
> That worked and flashed finally! But it doesn't change the icon for, I assume because I'm using a custom ROM with built-in customisation that modifies SystemUI. Anyhow, thanks for your all your work and support in trying to get this issue fixed. I doubt anything can be done with my current ROM so I'll just leave it at that.
> 
> ![Screenshot_20210210-120336](https://user-images.githubusercontent.com/45746576/107451974-7eb82680-6b9c-11eb-8c14-65203818fd0d.jpg)
> 
> ![Screenshot_20210210-120501](https://user-images.githubusercontent.com/45746576/107452041-a27b6c80-6b9c-11eb-8d8a-00e1f4720371.jpg)
> 
> ![Screenshot_20210210-121106](https://user-images.githubusercontent.com/45746576/107452052-a5765d00-6b9c-11eb-853a-5f4bb0f01d8c.jpg)
> 
> ![Screenshot_20210210-121113](https://user-images.githubusercontent.com/45746576/107452062-aa3b1100-6b9c-11eb-960e-1821de1bef45.jpg)

Sorry how did you get it to flash in the end?

**Comment by BilliTheKid** [comment]
> > > @wefalltomorrow Try to change `find /system` in customize.sh to `find / ` or `/product`
> > 
> > 
> > That worked and flashed finally! But it doesn't change the icon for, I assume because I'm using a custom ROM with built-in customisation that modifies SystemUI. Anyhow, thanks for your all your work and support in trying to get this issue fixed. I doubt anything can be done with my current ROM so I'll just leave it at that.
> > ![Screenshot_20210210-120336](https://user-images.githubusercontent.com/45746576/107451974-7eb82680-6b9c-11eb-8c14-65203818fd0d.jpg)
> > ![Screenshot_20210210-120501](https://user-images.githubusercontent.com/45746576/107452041-a27b6c80-6b9c-11eb-8d8a-00e1f4720371.jpg)
> > ![Screenshot_20210210-121106](https://user-images.githubusercontent.com/45746576/107452052-a5765d00-6b9c-11eb-853a-5f4bb0f01d8c.jpg)
> > ![Screenshot_20210210-121113](https://user-images.githubusercontent.com/45746576/107452062-aa3b1100-6b9c-11eb-960e-1821de1bef45.jpg)
> 
> No, this is actually a problem with magisk. Magisk cannot mount directories other than the system partition. I think it cannot support it.

I go this but then when I try to install it keeps saying failed. I'm on stock so I'm eager to see if it installs. How do I get it to install?
![Screenshot_20210210-160519](https://user-images.githubusercontent.com/56843472/107536662-edf04380-6bb9-11eb-924b-6c384234466e.png)
![Screenshot_20210210-160511](https://user-images.githubusercontent.com/56843472/107536663-ef217080-6bb9-11eb-8cec-b7faaf87a885.png)

**Comment by BilliTheKid** [comment]
> Oh that's a shame, thanks for the response and help trying to get this to work.

Can you tell me your steps that you got it to install in Magisk please?

**Comment by E7KMbb** [comment]
@BilliTheKid Change `find /system`  to  `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed

**Comment by BilliTheKid** [comment]
> @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed

Still getting an error
![Uploading Screenshot_20210210-235128.png…]()

**Comment by BilliTheKid** [comment]
> @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed

Still getting an error
![Screenshot_20210210-235151](https://user-images.githubusercontent.com/56843472/107588241-367c2100-6bfb-11eb-8226-2778eaa019e4.png)
![Screenshot_20210210-235128](https://user-images.githubusercontent.com/56843472/107588242-3714b780-6bfb-11eb-9b87-eea7c4068247.png)

**Comment by BilliTheKid** [comment]
> @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed

He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?

**Comment by E7KMbb** [comment]
> > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
>

> > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> 
> He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?

Find the directory where your systemui.apk is and change the directory in the script

**Comment by BilliTheKid** [comment]
> > > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
> 
> > > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> > 
> > 
> > He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?
> 
> Find the directory where your systemui.apk is and change the directory in the script

I still don't get it.
So what do I do from here?
![Screenshot_20210211-091451](https://user-images.githubusercontent.com/56843472/107618549-a74a2a00-6c49-11eb-9058-3facfff582f8.png)

**Comment by BilliTheKid** [comment]
> > > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
> 
> > > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> > 
> > 
> > He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?
> 
> Find the directory where your systemui.apk is and change the directory in the script

Mine is located at /system/system_ext/priv-app/SystemUIGoogle/SystemUIGoogle.apk

**Comment by E7KMbb** [comment]
> > > > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
> > 
> > 
> > > > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> > > 
> > > 
> > > He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?
> > 
> > 
> > Find the directory where your systemui.apk is and change the directory in the script
> 
> Mine is located at /system/system_ext/priv-app/SystemUIGoogle/SystemUIGoogle.apk

Let me check the results of these sh three

find /system -name SystemUI.apk
find /system -name *SystemUI.apk
find /system -name SystemUI*.apk

**Comment by BilliTheKid** [comment]
> > > > > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
> > > 
> > > 
> > > > > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> > > > 
> > > > 
> > > > He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?
> > > 
> > > 
> > > Find the directory where your systemui.apk is and change the directory in the script
> > 
> > 
> > Mine is located at /system/system_ext/priv-app/SystemUIGoogle/SystemUIGoogle.apk
> 
> Let me check the results of these sh three
> 
> find /system -name SystemUI.apk
> find /system -name _SystemUI.apk find /system -name SystemUI_.apk

![Screenshot_20210211-095616](https://user-images.githubusercontent.com/56843472/107622209-6bb25e80-6c4f-11eb-8483-9b0ca7e4bdd2.png)

**Comment by BilliTheKid** [comment]
> > > > > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
> > > 
> > > 
> > > > > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> > > > 
> > > > 
> > > > He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?
> > > 
> > > 
> > > Find the directory where your systemui.apk is and change the directory in the script
> > 
> > 
> > Mine is located at /system/system_ext/priv-app/SystemUIGoogle/SystemUIGoogle.apk
> 
> Let me check the results of these sh three
> 
> find /system -name SystemUI.apk
> find /system -name _SystemUI.apk find /system -name SystemUI_.apk

Can you explain to me on here what I need to write and change please as we are going round in circles

**Comment by E7KMbb** [comment]
> > > > > > @BilliTheKid更改`find /system`为`find /system_ext`，但是我想告诉您，有人尝试过此操作，但没有成功，无法解决此问题
> > > > 
> > > > 
> > > > > > @BilliTheKid Change `find /system` to `find /system_ext`, but I want to tell you that someone tried this but it didn’t work, this problem cannot be fixed
> > > > > 
> > > > > 
> > > > > He was on a custom ROM I'm on stock so I'd like to try it out. What am I doing wrong?
> > > > 
> > > > 
> > > > Find the directory where your systemui.apk is and change the directory in the script
> > > 
> > > 
> > > Mine is located at /system/system_ext/priv-app/SystemUIGoogle/SystemUIGoogle.apk
> > 
> > 
> > Let me check the results of these sh three
> > find /system -name SystemUI.apk
> > find /system -name _SystemUI.apk find /system -name SystemUI_.apk
> 
> Can you explain to me on here what I need to write and change please as we are going round in circles

Make changes based on the results,please execute them in the terminal

**Comment by BilliTheKid** [comment]
@E7KMbb What changes and how? When i ran the command nothing happened. I need a step by step on what i need to change and how please?

**Comment by E7KMbb** [comment]
> @E7KMbb What changes and how? When i ran the command nothing happened. I need a step by step on what i need to change and how please?

`find /system -name SystemUI*.apk` and `find /system -name *SystemUI.apk`

**Comment by BilliTheKid** [comment]
> > @E7KMbb What changes and how? When i ran the command nothing happened. I need a step by step on what i need to change and how please?
> 
> `find /system -name SystemUI*.apk` and `find /system -name *SystemUI.apk`

Ok done that, what next?
![Screenshot_20210211-134610](https://user-images.githubusercontent.com/56843472/107644704-ad530180-6c6f-11eb-8572-ec07bc43bb3f.png)

**Comment by E7KMbb** [comment]
> > > @E7KMbb What changes and how? When i ran the command nothing happened. I need a step by step on what i need to change and how please?
> > 
> > 
> > `find /system -name SystemUI*.apk` and `find /system -name *SystemUI.apk`
> 
> Ok done that, what next?
> ![Screenshot_20210211-134610](https://user-images.githubusercontent.com/56843472/107644704-ad530180-6c6f-11eb-8572-ec07bc43bb3f.png)

Are you really sure that your systemui.apk is in the system directory?  The code didn't find it

**Comment by BilliTheKid** [comment]
Is this the one your taking about?
![Screenshot_20210211-233432](https://user-images.githubusercontent.com/56843472/107712714-cd131580-6cc1-11eb-8e97-e8a000d32a07.png)

**Comment by BilliTheKid** [comment]
@E7KMbb 
![Uploading Screenshot_20210211-234929.png…]()

**Comment by BilliTheKid** [comment]
@E7KMbb 
![Screenshot_20210211-234929](https://user-images.githubusercontent.com/56843472/107713797-e321d580-6cc3-11eb-92e5-4fe3a3d95cc0.png)

**Comment by E7KMbb** [comment]
I released a new version, which may solve your problem

**Comment by wefalltomorrow** [comment]
[**@E7KMbb**](https://github.com/E7KMbb) commented on [28 Mar 2021, 7:02 pm AEDT](https://github.com/E7KMbb/Fake_5G_icon/issues/6#issuecomment-808862551 "2021-03-28T08:02:54Z - Replied by Github Reply Comments"):
> I released a new version, which may solve your problem

For both of us or just the OP?

Edit: I installed it without any errors but the icon didn't change sadly.

**Comment by BilliTheKid** [comment]
Yep loads in with no errors but it does not change the icon unfortunately.

**Comment by BilliTheKid** [comment]
> I released a new version, which may solve your problem

Icon didn't change.
